crds/virtualmachine.yaml

```diff
@@ -811,6 +811,11 @@ spec:
                   required:
                     - cores
                   properties:
+                    model:
+                      type: string
+                      default: "legacy"
```

Comment:
I feel that `legacy` is not a correct default here. It tells user nothing about real default model, i.e. "Nehalem"